components/sbm-recipes-jee-to-boot/src/test/java/org/springframework/sbm/jee/jaxrs/actions/ConvertJaxRsAnnotationsTest.java

```diff
@@ -16,12 +16,15 @@
 package org.springframework.sbm.jee.jaxrs.actions;
 
 import org.intellij.lang.annotations.Language;
+import org.junit.jupiter.api.Disabled;
+import org.junitpioneer.jupiter.ExpectedToFail;
```

Comment:
Was this `ExpectedToFail` import here added intentionally, and was the `@Disabled` usage below supposed to be `@ExpectedToFail`?

(because it appears currently this `ExpectedToFail` import is unused)